openpype/pipeline/colorspace.py

```diff
@@ -568,46 +617,61 @@ def get_labeled_colorspaces(
                 if color_data.get("aliases"):
                     aliases.update([
                         (
-                            alias_name,
+                            "aliases::{}".format(alias_name),
                             "[alias] {} ({})".format(alias_name, color_name)
                         )
                         for alias_name in color_data["aliases"]
                     ])
-                colorspaces.add(color_name)
+                colorspaces.add((
+                    "{}::{}".format(items_type, color_name),
+                    "[colorspace] {}".format(color_name)
+                ))
 
         elif items_type == "looks":
             looks.update([
-                (name, "[look] {} ({})".format(name, role_data["process_space"]))
+                (
+                    "{}::{}".format(items_type, name),
+                    "[look] {} ({})".format(name, role_data["process_space"])
+                )
                 for name, role_data in colorspace_items.items()
             ])
 
         elif items_type == "displays_views":
             display_views.update([
-                (name, "[view (display)] {}".format(name))
+                (
+                    "{}::{}".format(items_type, name),
+                    "[view (display)] {}".format(name)
+                )
                 for name, _ in colorspace_items.items()
             ])
 
         elif items_type == "roles":
             roles.update([
-                (name, "[role] {} ({})".format(name, role_data["colorspace"]))
+                (
+                    "{}::{}".format(items_type, name),
+                    "[role] {} ({})".format(name, role_data["colorspace"])
+                )
                 for name, role_data in colorspace_items.items()
             ])
```

Comment:
Also, doesn't it make sense to have the prefix be `role::` if the header is `[role]` instead of `roles::` just for consistency.

So:

- `role::` and `[role]`
- `look::` and `[look]`
- `alias::` and `[alias]`
- `colorspace::` and `[colorspace]`

The only outlier is `[view (display)]`  but also there it's likely even better to then also make that singular:

- `display_view::` and `[view (display)]

Reply:
Since the prefix is referring to the plural key in data but label makes sense to be singular, it would be better to keep it this way.

Reply:
To me it's clearer that that single entry refers to a `role::` type than it being part of the `roles::` group - but, whatever rocks your boat. I just got tripped up on them not matching from label to the data.